src/actions/vstudio/vs2005_csproj.lua

```diff
@@ -315,6 +315,45 @@
 			end
 		end)
 
+		if _ACTION >= "vs2010" and prj.nuget then
```

Comment:
This whole block should be split out into its own function so that it can be overridden in modules. Try to follow the model used in the vcxproj code: a function that uses `premake.callArray()` to output the individual elements. This whole C# exporter needs to be reworked to follow that model at some point; for now let's at least stick to it for any new code.